apps/chrome-devtools/src/app-devtools/theming-panel/common.ts

```diff
@@ -41,3 +42,106 @@ export const resolveVariable = (
   }
   return variableValue;
 };
+
+type Variant =
+  | '50'
+  | '100'
+  | '200'
+  | '300'
+  | '400'
+  | '500'
+  | '600'
+  | '700'
+  | '800'
+  | '900'
+  | 'A100'
+  | 'A200'
+  | 'A400'
+  | 'A700';
+
+const VARIANTS: Variant[] = [
```

Comment:
For this specific use case, wouldn't an Enum be more appropriate to make sure you do not forget any value in your list?
You could do 
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/29552488-ca19-436e-b7f6-1a0fdbe3a519)